components/autogen/src/gen-meta-support.py

```diff
@@ -56,6 +58,12 @@
 # ImagingEnvironment.Temperature = Partial see ticket #167 for details
 
 """
+OMEXML_VERSION = sys.argv[1]
+OMEXML_PATH = join(
+    expanduser("~"), ".m2", "repository", "org", "openmicroscopy", "ome-xml",
```

Comment:
This should likely be using the `maven.repo.local` ant property.  Couldn't the path to the jar be passed directly from ant?